drivers/sensors/nau7802.c

```diff
@@ -914,7 +914,7 @@ int nau7802_register(FAR struct i2c_master_s *i2c, int devno, uint8_t addr)
   priv->lower.ops = &g_sensor_ops;
   priv->lower.type = SENSOR_TYPE_FORCE;
   priv->enabled = false;
-  priv->odr = NAU7802_ODR_10HZ; /* 10Hz (0.1s) default ODR */
+  priv->odr = CONFIG_SENSORS_NAU7802_ODR;
```

Comment:
When the sensor boots, is it's default 10Hz? Because if so, you'll need to make sure that a user ODR of a different value is set on the chip, not just stored in the struct.

Also, make this configurable via the uORB interface for setting sampling frequency.